test/variable.jl

```diff
@@ -65,6 +65,14 @@ facts("[variable] get and set bounds") do
     @fact getUpper(fixedvar) --> 5
 end
 
+facts("[variable] get and set values") do
+    m = Model()
+    x0 = [1:3]
+    @defVar(m, x[1:3])
+    setValue(x,x0)
+    @fact getValue(x).innerArray --> x0
```

Comment:
I see from the build logs that getValue(x) returns an Array for that version of Julia+JuMP.

It runs fine on my setup, but maybe it's too outdated?
Julia 0.3.12-pre+3, JuMP 0.9.3